components/Image.tsx

```diff
@@ -101,4 +102,6 @@ const MyImage = React.forwardRef<HTMLImageElement, MyImageProps>(
   }
 );
 
+MyImage.displayName = "MyImage";
+
```

Comment:
What's this do?

Reply:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md